sopel/tools/jobs.py

```diff
@@ -179,6 +181,17 @@ def __init__(self, interval, func):
         self.interval = interval
         self.func = func
 
+    def is_ready_to_run(self, now):
+        """Tell if this job is ready to run at ``now`` time.
+
+        :param int now: Timestamp in seconds at which the job might be ready to
+                        run at
```

Comment:
Might be clearer to describe it like this:

```
:param int now: Timestamp to check, in seconds
```

Eliminates the duplicate "at"s, and leaves describing the purpose of the function to its description line and return value.

I understand why the timestamp is passed in instead of being fetched dynamically, but _because_ it's passed in, maybe "`now`" isn't the best name for the parameter?